components/admin_real/shared/constants.js

```diff
@@ -0,0 +1,2 @@
+export const dateFormat = "YYYY-MM-DD";
```

Comment:
its ok for now, but this is not ideal, this are constants for a specific component, and the file, its will be thought as constant for all components